Tilers/CityTiler/CityTilerDBConfigReference.yml

```diff
@@ -1,5 +1,5 @@
-PG_HOST: <FQDN or IP number of the server hosting your db>
-PG_PORT: <port>
-PG_NAME: <database name>
-PG_USER: <your db user name>
-PG_PASSWORD: <user password>
+PG_HOST: 172.18.0.1
```

Comment:
This file is a template file and should not be commited with
some specific configuration.

Reply:
fixed in d8dc6c3e4fc47f5b01b005b65ef4dfd02cfba0ed